packages/ripple-nuxt-tide/modules/webform/components/TideForm.vue

```diff
@@ -73,22 +73,34 @@ export default {
       const formData = this.formData.model
       const formId = this.formData.tideId
 
-      const res = await this.postForm(formId, formData)
-
-      if (res) {
-        this.formData.formState = {
-          response: {
-            status: 'success',
-            message: this.formData.messages.success || this.messages.success
+      if (formData.honeypot) {
+        // If the honeypot field has been filled, add delay and fake a success without further process.
+        setTimeout(() => {
+          this.formData.formState = {
+            response: {
+              status: 'other',
+              message: this.formData.messages.success || this.messages.success + ' (HP test)'
```

Comment:
Should `status: other` and `(HP test)` be output here? Or this this left-over code from testing?

I'm assuming the success message shown to honeypot captured submissions should be identical to the real success output message.

Reply:
@alan-cole yes, that's what I originally had. But I wanted to make it easy to test without any coding and checked the output. Nothing should really alert a bot that this was a fake success message.

But I'd also be fine removing the differences. I'll see what @dylankelly thinks.

Reply:
Removed text, left `status: other` for now.